src/Makefile.am

```diff
@@ -1,5 +1,5 @@
 AUTOMAKE_OPTIONS=foreign
-AM_CFLAGS = -I$(abs_top_srcdir)/include $(CODE_COVERAGE_CFLAGS)
+AM_CFLAGS = -I$(abs_top_srcdir)/include $(CODE_COVERAGE_CFLAGS) -fopenmp

```

Comment:
I think it should be an option for the configure script, you can refer to this:
https://github.com/lc-soft/LCUI/blob/460354c489c2ff97a77fd3c200f45c1efcf2e38a/configure.ac#L62-L75